robot/client/client.go

```diff
@@ -537,6 +537,12 @@ func (rc *RobotClient) connectWithLock(ctx context.Context) error {
 		} else {
 			err = multierr.Combine(err, grpcErr)
 		}
+
+		// instead of returning two context.DeadlineExceeded errors, tell the user how to fix
```

Comment:
```suggestion
		// A context.DeadlineExceeded from the WebRTC dial implies the client is unable to reach
		// the signaling server, which likely means that the client is offline. In that case, if the errors returned from grpc dials are also timeouts or mDNS failing to find a candidate, we should remind clients to double-check their internet connection and that the machine is on.
		// This should be more helpful than simply returning a chain of context.DeadlineExceeded and candidate not found errors.
```
since we're making a non-technical (product) based decision to hide/pretty drastically change error output, it is important to leave a useful comment so that future maintainers understand the reasoning behind these changes and can feel empowered to change the code if necessary